dialogflow/participant_management.py

```diff
@@ -127,7 +130,8 @@ def analyze_content_audio_stream(
     # After completing all of your requests, call the "__exit__()" method to safely
     # clean up any remaining background resources. Alternatively, use the client as
     # a context manager.
-    client = dialogflow.ParticipantsClient()
+    credentials, project_id = google.auth.default()
```

Comment:
In our local run, we usually just use `GOOGLE_APPLICATION_CREDENTIALS` environment variables to get the credentials. Is that something contradicting here?

Reply:
Nope, that is exactly right. This line will use those credentials. My issue was that the project defined in those credentials was a Free tier project, and not the same project as the Enterprise Tier project I as passing in as project_id. That conflict caused the behavior, so this change is an attempt to prevent that from happening for users in the future.